.ci/jobs/apm-agent-python-nightly-mbp.yml

```diff
@@ -24,6 +24,9 @@
           credentials: f6c7695a-671e-4f4f-a331-acdce44ff9ba
         build-strategies:
         - regular-branches: true
+        property-strategies:
+          all-branches:
+          - suppress-scm-triggering: true
```

Comment:
This will help to skip the builds upon merge to master events. See [docs](https://docs.openstack.org/infra/jenkins-job-builder/project_workflow_multibranch.html#project_multibranch.github_scm)